packages/microsoft_defender_endpoint/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -188,7 +188,7 @@ processors:
       field: json.description
       target_field: rule.description
       ignore_missing: true
-      if: (ctx.json?.description).length() < 1020
+      if: ctx?.json?.description != null && (ctx.json?.description).length() < 1020
```

```suggestion
      if: ctx.json?.description != null && ctx.json.description.length() < 1020
```

ctx is never null and we can remove `?` in second condition as null-check already done in first.